packages/charrua-core/charrua-core.0.1/opam

```diff
@@ -0,0 +1,16 @@
+opam-version: "1.2"
+name: "charrua-core"
+version: "0.1"
+maintainer: "Christiano F. Haesbaert <haesbaert@haesbaert.org>"
+authors: "Christiano F. Haesbaert <haesbaert@haesbaert.org>"
+homepage: "https://github.com/haesbaert/charrua-core"
+bug-reports: "https://github.com/haesbaert/charrua-core/issues"
+license: "ISC"
+dev-repo: "https://github.com/haesbaert/charrua-core.git"
+available: [ocaml-version >= "4.02"]
+build: [
+  ["sh" "build.sh"]
+]
+depends: [
```

Comment:
do you use the syntax extensions of anything?  That would also need camlp4 (but it built, so just checking in)